samples/RewriteSample/Startup.cs

```diff
@@ -11,14 +11,14 @@ namespace RewriteSample
 {
     public class Startup
     {
-        public void Configure(IApplicationBuilder app, IHostingEnvironment hostingEnvironment)
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
```

Comment:
Primary thinking for this was (1) some may want to pull the rewrite rules from web root instead of content root and (2) I'm starting to lay the foundation for configuring rewrites from `IConfiguration` and/or `Options`.